### PR TITLE
Unify blog and docs layout, styles

### PR DESCRIPTION
**Pull request by josadcha**
I have to add DocPage from Docusaurus to rewrite styles using module css
Blog sidebar doesn't have module css class, that's why the styles were applied directly to `aside` elem

### CODE REVIEW

File: src/theme/DocPage/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+@media (min-width: 1440px) {
+  :global(.docs-wrapper) {
+    max-width: var(--ifm-container-width-xl);
+  }
+}
+
+.docPage,
+.docMainContainer {
```
Comment (jengjeng) [inline src/theme/DocPage/styles.module.css]: just would like to know why it is camel case on the class name? or it is from docusaurus itself?

**Review by plsthink — CHANGES_REQUESTED** [review]
Issue with container width:
issue

File: src/theme/DocPage/styles.module.css (hunk 1 above)
Comment (plsthink) [reply src/theme/DocPage/styles.module.css]: These are css module style, used in in the component js file like an object, so better to use camel case here.

Comment (josadcha) [reply src/theme/DocPage/styles.module.css]: this is how you can name classes in module css
when page is rendered in browser you can see sth like this
image

**Review by kh-ruslan — APPROVED** [review]

File: src/theme/DocPage/styles.module.css (hunk 1 above)
Comment (plsthink) [reply src/theme/DocPage/styles.module.css]: I'm speaking of this: https://github.com/openware/yellow-org/blob/feature/unify-blog-docs/src/theme/DocPage/index.js#L22
And how it is used across the component, example:
https://github.com/openware/yellow-org/blob/feature/unify-blog-docs/src/theme/DocPage/index.js#L56

**Comment by plsthink** [comment]
Container width fix introduced a change on the width that was used before, please compare:
https://yellow-org.vercel.app/
https://yellow-org-t7kwx8pua-openware.vercel.app/

Which width is prefered for the actual design ?

File: src/theme/DocPage/styles.module.css (hunk 1 above)
Comment (jengjeng) [reply src/theme/DocPage/styles.module.css]: thanks. since we always use BEM in CSS files before. then it's clear

**Comment by josadcha** [comment]
> Container width fix introduced a change on the width that was used before, please compare:
> https://yellow-org.vercel.app/
> https://yellow-org-t7kwx8pua-openware.vercel.app/
> 
> Which width is prefered for the actual design ?

Locally I have the width like in master, checking what could become wrong

**Comment by plsthink** [comment]
Just saw that there is the same container width issue for the docs pages:
issue

**Comment by josadcha** [comment]
@lckmidnight thanks for noticing this bug. Now it looks good.

**Comment by plsthink** [comment]
> @lckmidnight thanks for noticing this bug. Now it looks good.

Preivous behavior was to have left sidebar for docs, stick to the left side, on wide screens, but it is not the case anymore.
Do we keep that behavior ?

issue

**Comment by josadcha** [comment]
@lckmidnight that was my decision in order to unify blog style and document portal style. For me it looks consistent because the container width in navbar is same on all pages - 1270px, so why not to have docs portal in the same width

Another solution would be to make blog wider, but it looks awful because blog has less content element in layout

**Comment by plsthink** [comment]
> @lckmidnight that was my decision in order to unify blog style and document portal style. For me it looks consistent because the container width in navbar is same on all pages - 1270px, so why not to have docs portal in the same width
> 
> Another solution would be to make blog wider, but it looks awful because blog has less content element in layout

Yes i understand the logic, but i think this may be an issue regarding docs readability, since we have a right sidebar for docs too.

If you compage docs where there are diagram, like here:
https://yellow-org.vercel.app/docs/concepts/smart-contracts
https://yellow-org-34jt4u3v0-openware.vercel.app/docs/concepts/smart-contracts

we can see that readability is affected
What do you think ?

**Comment by plsthink** [comment]
I think best solution is to keep actual width for landing and blog, but make an exceptioon for docs, as it behave in current master

**Comment by josadcha** [comment]
@lckmidnight  I got your point, I'll update PR later

**Review by plsthink — APPROVED** [review]
Perfect, good job!

**Pull request merged**